src/lib/style/Portal.scss

```diff
@@ -428,6 +428,15 @@ li.SRC-table-anchor-chosen, li.SRC-table-anchor-chosen > a {
    background-color: #D53F77 !important;
  }
 
+ .SRC-primary-background-color-hover:hover {
```

Comment:
Improve css selector specificity (usually I'd have a top level element id based selector, but you can't do that!) to remove all !important

Reply:
try specifying element (like body) instead (this may override style for Michael B)